zilencer/management/commands/sync_api_key.py

```diff
@@ -16,9 +16,12 @@ def handle(self, *args, **options):
         config_file = os.path.join(os.environ["HOME"], ".zuliprc")
         if not os.path.exists(config_file):
             raise RuntimeError("No ~/.zuliprc found")
-        config = SafeConfigParser()
+        config = ConfigParser()
         with open(config_file, 'r') as f:
-            config.readfp(f, config_file)
+            if hasattr(config, 'read_file'):
+                config.read_file(f, config_file)
+            else:
+                config.readfp(f, config_file)
```

Comment:
Do we actually need the `else` clause here?  I would just eliminate it, since the "six" library presumably always gives us the same API.  I think it's generally good to write defensive code, but we should keep things simple here.  If I'm wrong, it's only failing on parsing configs, so there won't be too much risk here.